src/App.scss

```diff
@@ -59,3 +59,7 @@ body {
 .App__command-palette-command {
     margin-top: 1rem;
 }
+
+:export {
+    characterMovementLength: $c2lc-scene-tile-width;
```

Comment:
This is a tricky one, but I think it's slightly preferable here to keep the existing name of `sceneGridCellWidth`, rather than introducing a new name.